ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/GAnalytics.h

```diff
@@ -0,0 +1,53 @@
+#ifndef GANALYTICS_H
+#define GANALYTICS_H
+
+#include <QDateTime>
+#include <QObject>
+#include <QVariantMap>
+
+class GAnalytics : public QObject
+{
+  Q_OBJECT
+
+  Q_PROPERTY(QString clientId READ clientId CONSTANT)
+  Q_PROPERTY(QString apiSecret READ apiSecret WRITE setApiSecret NOTIFY apiSecretChanged)
```

Comment:
🙃 For some of these boiler plate properties where you are just getting and setting the member, you can use the MEMBER keyword and remove the getters/setters all together

```suggestion
  Q_PROPERTY(QString apiSecret MEMBER m_apiSecret NOTIFY apiSecretChanged)
```